### ISSUE TITLE
Why do you use cosine weight decay?

### ISSUE THREAD

**Issue by CoinCheung** [pengzhiliang/MAE-pytorch#38]
Hi,

Thanks for providing the code. I found that you are also using cosine annealing curve to decay weight decay. It seems that the paper does not mention this. Would you please tell me why do you use it ?

**Comment by pengzhiliang** [comment]
Hello, follow BEiT, I implement the cosine weight decay strategy in this repo. However, I didn't use it.

You can check the `"train_weight_decay"` item in [log](https://github.com/pengzhiliang/MAE-pytorch/blob/main/files/pretrain_base_0.75_400e.txt)
